src/Haskell/Ide/Engine/Plugin/Liquid.hs

```diff
@@ -120,9 +120,10 @@ diagnosticProvider DiagnosticOnSave uri cb = pluginGetFile "Liquid.diagnosticPro
     LiquidData mtid <- get
     mapM_ (liftIO . cancel) mtid
 
-    let progTitle = "Running Liquid Haskell on " <> T.pack (takeFileName file)
-    tid <- withIndefiniteProgress progTitle $
-      liftIO $ async $ generateDiagnosics cb uri file
+    -- let progTitle = "Running Liquid Haskell on " <> T.pack (takeFileName file)
+  --  tid <- async $ withIndefiniteProgress progTitle $ (liftIO $ generateDiagnosics cb uri file)
```

Comment:
What's going on here?

Reply:
Should be resolved now, needed to make `withIndefiniteProgressIO`